src/main/java/at/hannibal2/skyhanni/features/inventory/bazaar/BazaarApi.kt

```diff
@@ -164,8 +166,16 @@ object BazaarApi {
         return NeuInternalName.fromItemName(bazaarItem.displayName)
     }
 
+    private var disabledApiJson: DisabledApiJson? = null
+
+    @HandleEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        disabledApiJson = event.getConstant<DisabledApiJson>("misc/DisabledApi")
```

Comment:
instead of duplicating this repo load three times, write this once into a new utils class (or inside the api class), then access this class in the three other classes